client_test.go

```diff
@@ -0,0 +1,244 @@
+package luasandbox_test
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/calyptia/api/types"
+	"github.com/calyptia/lua-sandbox-client"
+)
+
+func shouldError(t *testing.T, expected string, err error) {
+	if err == nil {
+		t.Errorf("Expected error: %v", expected)
+		return
+	}
+
+	if expected != err.Error() {
+		t.Errorf("Expected error: %v, got: %v", expected, err.Error())
+	}
+}
+
+func shouldEqual(t *testing.T, expected, actual []luasandbox.LogResult) {
```

Comment:
There is an assert library we have been using `github.com/alecthomas/assert/v2` that has these two methods:

- `assert.Equal()`
- `assert.EqualError()`

Feel free to `go get` it since test dependencies are not compiled and don't suppose extra weight ;)

Reply:
Why is it bad adding it to `go.mod` ? Are test dependencies compiled in prod for library users?

Reply:
I didn't say it is bad. Please do 👍